### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KatyushaScarlet**
Add pre-build steps

### CODE REVIEW

**Comment by KatyushaScarlet** [comment]
Hi, I am trying to compile the buildroot of RG350 on CentOS 7 and Debian Buster. This is what I have summarized from the old README and the rg350_buildroot_docker repository of soarqin, and it has been corrected and verified.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+~~~bash
+sudo apt-get update -y
+sudo apt-get install -y bison flex gettext texinfo wget cpio mercurial subversion libncurses5-dev libc6-dev-i386 bzr squashfs-tools zip unzip python python3 rsync
+~~~
+
+Or if you are using CentOS, please run:
+
+~~~bash
+yum update -y
+yum install -y autoconf automake bc bison bzip2 flex fontconfig freetype gcc-c++ git glibc-devel glibc-devel.i686 java-1.8.0-openjdk-devel libgcc.i686 libstdc++.i686 m4 make mercurial openssl-devel patch perl-ExtUtils-MakeMaker rsync squashfs-tools subversion zip unzip wget which python3 texinfo
```
Comment (glebm) [inline README.md:17]: jamvm is not in the buildroot anymore so I think java-1.8.0-openjdk-devel is not needed

Comment (KatyushaScarlet) [reply README.md:17]: Sorry, I omitted that jamvm has been removed from buildroot.

**Comment by glebm** [comment]
@KatyushaScarlet By the way, you may want to use this buildroot instead:

https://github.com/od-contrib/buildroot-rg350-old-kernel/releases

**Comment by KatyushaScarlet** [comment]
> @KatyushaScarlet By the way, you may want to use this buildroot instead:
> 
> https://github.com/od-contrib/buildroot-rg350-old-kernel/releases

Will [this](https://github.com/tonyjih/RG350_buildroot) repository be abandoned or it is just for archiving?

**Comment by glebm** [comment]
This buildroot is old userland (with some updates) + old kernel, it should probably stay this way.

The other repo is new userland + old kernel, I think it's better to focus our efforts there.

**Pull request closed without merging**